projects/@shared/lib/validator/extensions/words.js

```diff
@@ -34,11 +35,35 @@ const words = joi => ({
         const trimmedValue = (value && value.length) ? value.trim() : value;
         const array = trimmedValue.length ? trimmedValue.split(' ') : trimmedValue;
 
+        if (array && array.length < args.words) {
+          return { value, warn: helpers.warn('words.minWords', { words: args.words }) };
+        }
+
+        return value;
+      },
+    },
+    maxWords: {
+      alias: 'max',
+      method(words) {
+        return this.$_addRule({ name: 'maxWords', args: { words } });
+      },
+      args: [
+        {
+          name: 'words',
+          ref: true,
+          assert: value => typeof value === 'number' && !isNaN(value),
+          message: 'must be a number',
+        },
+      ],
+      validate(value, helpers, args) {
+        const trimmedValue = (value && value.length) ? value.trim() : value;
```

Comment:
Ook deze kun je schrijven als `?.`. (Edit: Ik zie dat je dit op meerdere plekken hebt gedaan, en dus zal ik het niet overal markeren. Dit hoef je btw niet te "fixen" het is maar een suggesties.)